### DISCUSSION TITLE
Broadcast screen does not work

### DISCUSSION THREAD

**Discussion by XanthiM** [epoptes/epoptes#70] [Q&A]
Hello. 
I have installed epoptes 0.5.10 in Linux Mint 19. Everything works find except screen broadcasting. 
What should i do?

**Comment by alkisg** [comment]
Assuming Linux Mint 19 is based on Ubuntu 18.04, see this reply:
https://github.com/Epoptes/epoptes/issues/72#issuecomment-422350112
